src/braket/experimental/autoqasm/api.py

```diff
@@ -17,7 +17,7 @@
 import functools
 import inspect
 from types import FunctionType
-from typing import Any, Callable, Dict, List, Optional, Tuple, Union
+from typing import Any, Callable, Optional, Union
```

Comment:
You can also use the non-`typing` `Callable` now:
```python
from collections.abc import Callable
```

Reply:
Okay cool! I had also had a version of this that removed Option and Union before realizing that is only 3.10+ 😅